tests/CMakeLists.txt

```diff
@@ -60,6 +60,19 @@ add_test(NAME repro_avif_decode_fuzzer COMMAND repro_avif_decode_fuzzer
 ################################################################################
 # GoogleTest
 
+# Adds a gtest from file TEST_NAME.cc located in the gtest folder. Extra arguments
+# are considered as extra linked libraries.
+macro(add_avif_gtest TEST_NAME USE_DATA)
+    add_executable(${TEST_NAME} gtest/${TEST_NAME}.cc)
+    target_include_directories(${TEST_NAME} PRIVATE ${GTEST_INCLUDE_DIRS})
+    target_link_libraries(${TEST_NAME} PRIVATE aviftest_helpers ${GTEST_BOTH_LIBRARIES} ${ARGN})
+    if(${USE_DATA})
+        add_test(NAME ${TEST_NAME} COMMAND ${TEST_NAME} ${CMAKE_CURRENT_SOURCE_DIR}/data/)
```

Comment:
I wonder if we cannot pass `${CMAKE_CURRENT_SOURCE_DIR}/data/` everywhere, to simplify. Maybe it will make the default gunit_main handler fail because of an unrecognized flag.

Otherwise maybe replace `0` and `1` by `USE_DATA` or something more explicit below.

Reply:
I created 2 macros